app/konnect/configure/runtime-manager/runtime-groups/declarative-config.md

```diff
@@ -133,10 +153,9 @@ the example with your generated UUID:
     ```sh
     creating service 9595B5F9-3B6A-4C48-BE93-9EC1B0EA487A
```

Comment:
Question: What is this service for?

I don't think this is part of the config. (especially if we're not specifying any `{YOUR_GENERATED_UUID}`s.

Reply:
Nvm, I think that's what your next comment is pointing out.